network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -1297,6 +1296,7 @@ def categories_page(self, request, slug):
         context['category'] = category.slug
         context['products'] = products
         context['pageTitle'] = f'{category} | ' + gettext("Privacy & security guide") + ' | Mozilla Foundation'
+        context['template_cache_key'] = category.slug
```

Comment:
this will break later when we switch to "literally different pages for different locales", let's work the locale into the key even if with wagtail-modeltranslation that isn't a problem yet.